src/librustc_back/target/windows_base.rs

```diff
@@ -63,7 +60,32 @@ pub fn opts() -> TargetOptions {
 
             // Always enable DEP (NX bit) when it is available
             "-Wl,--nxcompat".to_string(),
+
+            // Do not use the standard system startup files or libraries when linking
+            "-nostdlib".to_string(),
```

Comment:
Was this necessary? I thought that `-nodefaultlibs` would do what this comment indicates, and the man pages at least indicate that `-nostdlib` means that only the directories on the command line will be searched (e.g. doesn't mention anything about startup files or objects).